### DISCUSSION TITLE
Merging databases

### DISCUSSION THREAD

**Discussion by jackwilsdon** [krateng/maloja#385] [Q&A] [answered]
I've got two databases from two instances of Maloja, each with a number of scrobbles, and I'd like to merge them into a single database. Is there an easy way to do this?

**Comment by jan666** [comment, marked as answer]
I have not tested this but I guess `maloja export` on instance a followed by `maloja import` on instance b should work

**Reply by jackwilsdon** [reply]
Aha, I didn't realise maloja had export/import commands. Thanks!
